recipes/portaudio/meta.yaml

```diff
@@ -0,0 +1,32 @@
+{% set version = "190600_20161030" %}
```

Comment:
Could you change the version number to `19.6.0`, to match defaults, please?

Reply:
So defaults is not following that ugly (but original) version string :stuck_out_tongue_winking_eye:

Reply:
I don't know what the history of this is on defaults.  We only have Mac builds right now.  I'd be OK if we agreed on either way, but I will say that conda-build 3 will work much better with the semver-style adapted version.  Conda probably will do better with it, too, though it should be able to cope either way.

Reply:
Thanks @msarahan. For now let's go with what is already there. I modified it to be `19.6.0`.